app/api-requests/third-party-api/person.js

```diff
@@ -0,0 +1,27 @@
+const { get } = require('./base')
+const session = require('../../session')
+const { tokens } = require('../../session/keys')
+const config = require('../../config')
+const jwtDecode = require('../../auth/token-verify/jwt-decode')
+const hostname = config.authConfig.ruralPaymentsAgency.hostname
+const getPersonSummaryUrl = config.authConfig.ruralPaymentsAgency.getPersonSummaryUrl
+
+function getPersonName (personSummary) {
+  return [personSummary.firstName, personSummary.middleName, personSummary.lastName].filter(Boolean).join(' ')
+}
+
+function parsedAccessToken (request) {
+  const accessToken = session.getToken(request, tokens.accessToken)
+  return jwtDecode(accessToken)
+}
+
+const getPersonSummary = async (request) => {

```

Comment:
could we add a code comment to mention the hard coded personId please?

Reply:
Comment added